### ISSUE TITLE
[tonlib] Error on init

### ISSUE THREAD

**Issue by akme** [ton-blockchain/ton#286]
After last commit tonlib fails to start with an error:
```
[ 0][t 1][1587996748.677032232][Status.h:571][!Tonlib][&status_.is_ok()]	[Error : 0 : Wrong character in the string]
```

**Comment by Rexagon** [comment]
Same here. I think the problem is in [**this**](https://github.com/ton-blockchain/ton/blob/master/tonlib/tonlib/TonlibClient.cpp#L1932) invalid hash

**Comment by vp-mazekine** [comment]
+1

**Comment by nickdndev** [comment]
+1

**Comment by aleksandr-hramcov** [comment]
+1

**Comment by rkorotchin** [comment]
The same for me.

**Comment by rkorotchin** [comment]
+1

**Comment by Skydev0h** [comment]
> Same here. I think the problem is in [**this**](https://github.com/ton-blockchain/ton/blob/master/tonlib/tonlib/TonlibClient.cpp#L1932) invalid hash

And SUDDENLY this points out to a **mainnet** configuration that already has preset **root_hash** and **file_hash** (although incorrectly formatted, maybe in a hurry)... _Conspiracy time!_

**Comment by akme** [comment]
Fixed with eecf05c commit.
